build.gradle

```diff
@@ -182,12 +182,14 @@ subprojects {
             bouncycastle: 'org.bouncycastle:bcpkix-jdk15on:1.67',
 
             // Test dependencies.
-            junit: 'junit:junit:4.12',
+            junit: 'junit:junit:4.13',
             mockito: 'org.mockito:mockito-core:3.3.3',
             truth: 'com.google.truth:truth:1.0.1',
             guava_testlib: "com.google.guava:guava-testlib:${guavaVersion}",
+            androidx_annotation: "androidx.annotation:annotation:1.1.0",
+            androidx_core: "androidx.core:core:1.3.0",
             androidx_test: "androidx.test:core:1.3.0",
-            robolectric: "org.robolectric:robolectric:4.3.1",
+            robolectric: "org.robolectric:robolectric:4.4",
```

Comment:
I tried to bump this dependency in https://github.com/grpc/grpc-java/pull/7733, but ran into https://github.com/grpc/grpc-java/issues/7731.

Reply:
Added a comment there, I think that should be resolved with the switch to API 29.